### DISCUSSION TITLE
'overflow-x-auto' - this property doesn’t work in iOS browsers (Chrome/Safari)

### DISCUSSION THREAD

**Discussion by Medkhat** [tailwindlabs/tailwindcss#9512] [Help]
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/42961493/194740070-c36dcf93-ca0c-4525-8a20-e9898cd33bbf.png">

Hi everyone! Thank you for such a cool CSS tool! I couldn't figure out what the problem is. Maybe someone knows about it. Thank you a lot!

**Comment by KevinHoppeLS** [comment]
Hey! I remember fixing some scroll behaviour issues on IOS using following CSS code block:

```
@supports (-webkit-touch-callout: none) {
        body {
            height: -webkit-fill-available;
        }
        html {
            height: -webkit-fill-available;
        }
    }
```

Maybe give it a shot?
